android/app/src/main/java/com/expensify/chat/customairshipextender/CustomNotificationProvider.java

```diff
@@ -171,10 +171,7 @@ private void applyMessageStyle(@NonNull Context context, NotificationCompat.Buil
         String message = reportAction.get("message").getList().get(0).getMap().get("text").getString();
         long time = Timestamp.valueOf(reportAction.get("created").getString(Instant.now().toString())).getTime();
         String roomName = payload.get("roomName") == null ? "" : payload.get("roomName").getString("");
-        String conversationTitle = "Chat with " + name;
-        if (!roomName.isEmpty()) {
-            conversationTitle = "#" + roomName;
-        }
+        String conversationTitle = roomName.isEmpty() ? "Chat with " + name : roomName;
```

Comment:
I think we should simply delegate the room naming convention to the server. Else we'd have to duplicate this logic to each native platform.

Reply:
This might be unrelated to this PR, I was just wondering if we have in plan to add i18n in the native code as well, @Julesssss ? It this case seems that the notifications have english title even if the user has Spanish as the preferred language.

Reply:
Good question, I have no idea. Is that something we can handle from the back end?

Reply:
So yeah I sort of agree? See this conversation though in Slack - https://expensify.slack.com/archives/C049HHMV9SM/p1675375243768159

I think the challenge right now is that the server just isn't internationalized at all, so there's no framework for it. I think the idea for now is that we do this now and update down the road?

Reply:
Agree that this can be done separately